tests/cloner_test.go

```diff
@@ -118,9 +118,6 @@ var _ = Describe("all clone tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 			f.ForceBindPvcIfDvIsWaitForFirstConsumer(targetDataVolume)
 
-			By("Wait for clone to be completed")
-			_, err = utils.WaitForPVC(f.K8sClient, targetDataVolume.Namespace, targetDataVolume.Name)
-			Expect(err).ToNot(HaveOccurred())
```

Comment:
Maybe it is not needed, since we already wait for DV.

But I think it has nothing to do with force binding.So I do not understand your commit message. Force Bind only works for WFFC storage, and when run on Immediate storage the function forceBind works like a noop.

Reply:
anyway, this comments is only about a commit message that is not adequate to change